zcash_extensions/Cargo.toml

```diff
@@ -15,6 +15,7 @@ zcash_primitives = { version = "0.11", path = "../zcash_primitives", default-fea
 
 [dev-dependencies]
 ff = "0.13"
+incrementalmerkletree = { version = "0.3", features = ["legacy-api"] }
```

Comment:
```suggestion
```

Reply:
And if so, this can be removed.

Reply:
This cannot be removed, because it's still used for tests.